### PR TITLE
Created two method inside the new interface for more easily code unde…

### PR DESCRIPTION
**Pull request by laracoo**
…rstand + make code more clean and right

### CODE REVIEW

File: src/main/java/com/jadventure/game/monsters/Bugbear.java [hunk 1]
```diff
@@ -7,17 +7,22 @@
  */
 public class Bugbear extends Monster {
 	public Bugbear(int playerLevel){
-		this.monsterType = "Bugbear";
+		monsterType("Bugbear");
```
Comment (Dev-Osmium) [inline src/main/java/com/jadventure/game/monsters/Bugbear.java]: I don't quite get the purpose of removing this single line of code into a method.

File: src/main/java/com/jadventure/game/monsters/CreateNonster.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package com.jadventure.game.monsters;
+
+/**
+ * The interface for two methods for realise the all monsters inside this class
+ */
+public interface CreateNonster {
```
Comment (Dev-Osmium) [inline src/main/java/com/jadventure/game/monsters/CreateNonster.java]: I'm assuming that you meant `CreateMonster` here

**Comment by paddatrapper** [comment]
Thanks for the PR! Can you please explain in more detail why you propose this change?

**Comment by laracoo** [comment]
Hello, I am just a beginner programmer and I really liked this project. With this code I wanted to simplify the work of the program, as it seems to me! Since I am just starting, I have many mistakes, but a wild desire to participate! I would be very happy if you would give some tips or explanations, thanks in advance🙏🏻

**Comment by paddatrapper** [comment]
> Hello, I am just a beginner programmer and I really liked this project.

That is great! Welcome

> With this code I wanted to simplify the work of the program, as it seems to me! 

Ok, how does it simplify the work of the program?

**Comment by Dev-Osmium** [comment]
@paddatrapper Thanks for clarifying my comments

**Comment by Dev-Osmium** [comment]
@paddatrapper Thanks for clarifying my comments.

**Pull request closed without merging**